src/content/en/2021/pwa.md

```diff
@@ -17,4 +17,597 @@ featured_stat_3: TODO
+  description='Time series of page loads from 1st January 2019 until September 2021 showing a growth from 1% to just over 4% with a large growth in January 2021.',
+  width=1460,
+  height=786
+  )
+}}
+
+According to <a hreflang="en" href="https://www.chromestatus.com/metrics/feature/timeline/popularity/1436">Chrome Platform Status</a>, the percentage of page loads using this feature <a hreflang="en" href="https://www.chromestatus.com/metrics/feature/timeline/popularity/1436">is near 4%</a>, which suggests that some high traffic sites might be using it. Additionally, we can see that there was a 2.5 percentage point growth in adoption compared to last year.
+
+### App Store distribution
+
+Historically, developers have built web-based mobile applications and uploaded them to App Stores as an alternative to building apps with OS-specific languages (Java or Kotlin for Android, Objective-C or Swift for iOS). The most common approach is to use a cross-platform, hybrid solution like <a hreflang="en" href="https://cordova.apache.org/">Cordova</a> that allows one to write the code once and generate multiple versions of it for various platforms. The resulting code usually uses the <a hreflang="en" href="https://developer.android.com/reference/android/webkit/WebView">WebView</a> to render web content, but also provides a series of non-standard APIs that can access features from the device.
+
+WebView-based apps may look similar to native apps, but certainly there are some caveats. Since a WebView is just a rendering engine, users may have different experiences than in a full browser. The latest browser APIs might not be available and most importantly, cookies are not shareable between WebViews and browsers.
+
+TWAs allow you to package your PWA into a native application shell and upload it to some App Stores.
+Unlike WebView-based solutions, a TWA is not just a rendering engine; it's the full browser running in fullscreen mode. For that reason it's feature-complete and evergreen, meaning that it's always up to date and will give you access to the latest web APIs.
+
+Developers can package their PWAs into native apps with TWA directly, <a hreflang="en" href="https://developer.chrome.com/docs/android/trusted-web-activity/integration-guide/">by using Android Studio</a>, but there are several tools that make this task much easier. Next we'll analyze two of them: PWA Builder and Bubblewrap.
+
+#### PWA Builder
+
+<a hreflang="en" href="https://www.pwabuilder.com/">PWA Builder</a> is an open-source project that can help web developers to build Progressive Web Apps and package them for app stores like the Microsoft Store and Google Play Store. It starts by reviewing a provided URL to check for an available manifest, service worker, and SSL.
+At the moment of this writing (October 2021) <a hreflang="en" href="https://twitter.com/pwabuilder/status/1454250060326318082?s=21">on the 200k last URLs reviewed (3 months' timeslot) by PWA Builder</a>:
```

Comment:
Newline. Plus let's try to avoid using October when all the other stats are July.

```suggestion

<a hreflang="en" href="https://twitter.com/pwabuilder/status/1454250060326318082?s=21">PWA Builder reviewed 200k URLs over a 3 month timeslot</a> and discovered that:
```